modules/ROOT/pages/setup.adoc

```diff
@@ -99,7 +100,7 @@ wrapper.java.maxmemory=3024
 
 === Setup the Payload Storage API
```

Comment:
GLOBAL: Search for "setup" and evaluate for correct presentation.

setup = noun, adjective
set up = verb